wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/functions.php

```diff
@@ -203,3 +230,104 @@ function wporg_ov_customizer( $wp_customize ) {
 	) );
 }
 add_action( 'customize_register', __NAMESPACE__ . '\wporg_ov_customizer' );
+
+/*
+	=====================================
+	Openverse standalone site redirection
+	=====================================
+ */
+
+/**
+ * This is the URL at which the standalone Openverse site is hosted. When
+ * redirect is enabled (see setting `ov_is_redirect_enabled`), the theme
+ * redirects all incoming requests to the right URL on this domain.
+ *
+ * Note: Do not put a trailing slash '/' in this URL, as it will cause problems.
+ */
+if ( !defined( 'OPENVERSE_STANDALONE_URL' ) ) {
+	define( 'OPENVERSE_STANDALONE_URL', 'https://openverse.wordpress.net' );
+}
+
+/**
+ * Determine the target URL of the redirect based on the Openverse standalone
+ * URL, the requested path and the current locale.
+ *
+ * Examples:
+ * - https://ru.wordpress.org/openverse → {OPENVERSE_STANDALONE_URL}/ru/
+ * - https://wordpress.org/openverse/search/?q=dog → {OPENVERSE_STANDALONE_URL}/search/?q=dog
+ */
+function get_target_url() {
+	$target_url = OPENVERSE_STANDALONE_URL;
+
+	$curr_locale = get_locale();
+	$locale = get_locale_slug( $curr_locale );
+	if ( $locale !== '' ) {
+		$target_url .= '/' . $locale;
+	}
+
+	$path = $_SERVER['REQUEST_URI'];
+	if ( $path ) {
+		$count = 1; // Only replace the leading Openverse subpath.
+		$target_url .= str_replace( OPENVERSE_SUBPATH, '', $path, $count );
+	}
+
+	return $target_url;
+}
+
+/**
+* Provide configuration for the theme to redirect to the given standalone
+* Openverse site. The destination URL can be configured and the behaviour can
+* be dormant unless enabled.
+*
+* @param \WP_Customize_Manager $wp_customize Theme Customizer object.
+*/
+function wporg_ov_redir_customizer( $wp_customize ) {
+	$wp_customize->add_section( 'ov_redir', array(
+		'priority' => 10,
+		'capability' => 'edit_theme_options',
+		'title' => 'Openverse Redirect',
+		'description' => 'Configure the redirection to the standalone Openverse site.'
+	) );
+
+	$wp_customize->add_setting( 'ov_redirect_url', array(
+		'type' => 'theme_mod',
+		'capability' => 'edit_theme_options',
+		'default' => OPENVERSE_STANDALONE_URL,
```

Comment:
Here we set the default value of the `ov_redirect_url` to the `OPENVERSE_STANDALONE_URL` constant.